x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/constants.ts

```diff
@@ -34,3 +34,8 @@ export const DEFAULT_INFERENCE_ENDPOINTS_TABLE_STATE: AllInferenceEndpointsTable
 };
 
 export const PIPELINE_URL = 'ingest/ingest_pipelines';
+
+export const PRECONFIGURED_ENDPOINTS = {
+  ELSER: '.elser-2',
```

Comment:
These endpoints are renamed in https://github.com/elastic/elasticsearch/pull/115395